src/js/plugins/plugin.resume.js

```diff
@@ -0,0 +1,88 @@
+/* global BookReader */
+
+/**
+ * Plugin to remember the current page number in a cookie
+ */
+jQuery.extend(BookReader.defaultOptions, {
+  enablePageResume: true,
+  /**
+   * @const null|string eg '/', '/details/id'
+   */
```

Comment:
Here's the correct way to type this; you can check if the jsdoc is correct by hovering over `resumeCookiePath` in vscode, and seeing if it got the right type. 

```suggestion
  /** @type {string|null} eg '/', '/details/id' */
```